### PR TITLE
turn markers into a real property

### PR DESCRIPTION
**Pull request by jakemac53**
closes https://github.com/GoogleWebComponents/google-map/issues/183

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 bower_components
+.idea
```
Comment (ebidel) [inline .gitignore]: i hate new ideas!

File: google-map.html [hunk 2]
```diff
@@ -331,6 +331,14 @@
...
         type: Object,
         value: function() { return {}; }
+      },
+
+      /**
+       * The markers on the map.
+       */
+      markers: {
+        type: Array,
+        value: function() { return []; }
```
Comment (ebidel) [inline google-map.html]: we should make this `readOnly: true`. That means you'll need to use the magic `this._setMarkers(val)`

**Comment by jakemac53** [comment]
updated/squashed

**Comment by ebidel** [comment]
:+1:

**Pull request merged**